services/spar/src/Spar/Data.hs

```diff
@@ -444,17 +444,17 @@ getDefaultSSOCode = fmap runIdentity . minimumMay <$>
   (retry x1 . query sel $ params Quorum ())
   where
     sel :: PrepQuery R () (Identity SAML.IdPId)
-    sel = "SELECT idp FROM default_idp WHERE partition_key_always_default = 'default'"
+    sel = "SELECT idp FROM default_idp WHERE partition_key_always_default = 'default' ORDER BY idp"
```

Comment:
I think if you make this `DESC`, it might even always give you the *newer* entry.  Not sure how uuids work, though.

Reply:
I didn't assume any meaningful ordering in the UUIDs (and I don't think they have one), I just want that there is *some* ordering, so running the same query twice is guaranteed to give the same result.

Reply:
looks like it's ok to assume an ordering, though:

```
$ uuid ; uuid ; uuid ; uuid ; uuid ; uuid ; uuid ; uuid ; uuid ; uuid
b93369c6-4697-11ea-ae1a-27c8e8d1ded5
b9339b58-4697-11ea-8ed2-abf4b3b6a750
b933c588-4697-11ea-8060-8f056ba4913b
b933ee50-4697-11ea-96a9-37a65b436789
b9341a1a-4697-11ea-b801-3fb0f1caecfc
b9344a62-4697-11ea-a810-1ff327e8656e
b934797e-4697-11ea-89dc-a3580b24d3b4
b934a7d2-4697-11ea-8022-9b26ab5b572d
b934d6ee-4697-11ea-9799-7fbf7c802f05
b93525e0-4697-11ea-aacf-a32eda12db52
```

this suggests descending ordering would always give the most --

ah, never mind, i was confusing the time at which an idp is made the default with the time at which it is created.

i would still make `DESC` since it works better in the more common case that a new idp has just been created an then made the default.